README.md

```diff
@@ -114,7 +114,7 @@ export class AppModule {}
 
 Let's explain each one of the `config` options:
 
-- `reRenderOnLangChange`: Subrscribes to the language change event, and allows you to change the active language. This is not needed in applications that don't allow the user to change the language in runtime (i.e., from a dropdown), so by setting it to false in these cases, you can save on memory by rendering the view once, and unsubscribing from the language changes event (defaults to `false`).
+- `renderLangOnce`: Subscribes to the language change event, and allows you to change the active language. This is not needed in applications that don't allow the user to change the language in runtime (i.e., from a dropdown), so by setting it to false in these cases, you can save on memory by rendering the view once, and unsubscribing from the language changes event (defaults to `false`).
```

Comment:
@NetanelBasal I probably made a mistake when rebasing, the correct opt is `reRenderOnLangChange` ?

Reply:
Yes